modules/backup-and-restore/pages/configurations.adoc

```diff
@@ -48,7 +48,14 @@ Typically used when operating in a private network or with a non-AWS S3-compatib
 |System.Backup.CompressProcessNumber | Number of concurrent processes for compression during backup.
 
 It's recommended to keep the default value `0`, which means the number of processes used to compress is equal to the number of CPU cores on each node.
-| `0`
+| `8`
+
+|System.Backup.DecompressProcessNumber | The number of concurrent processes for decompression during the restore.
+| `8`
+
+|System.Backup.CompressionLevel |The backup compression level strikes a balance between size and speed. The better compression, the longer it takes.
+("BestSpeed", "DefaultCompression", "BestCompression")
+| `DefaultCompression`
```

Comment:
```suggestion
|System.Backup.CompressionLevel
|One of three present levels of compression.  Higher compression takes longer. 
("BestSpeed", "DefaultCompression", "BestCompression")
| "DefaultCompression"
```

Reply:
The https://docs.tigergraph.com/tigergraph-server/current/reference/configuration-parameters is supposed to list ALL the configuration parameters.  This page (https://docs.tigergraph.com/tigergraph-server/current/backup-and-restore/configurations) may repeat a subset of those confirmation parameters, for convenience.

So, any change you make should me made in both places.  Please double check them for consistency.

Reply:
Thank you @victorleeTG I made these changes with my last commit.